mithril-aggregator/src/tools/certificates_hash_migrator.rs

```diff
@@ -244,7 +247,8 @@ mod test {
         for (certificate, discriminant_maybe) in certificates_and_signed_entity {
             certificate_repository
                 .create_certificate(certificate.clone())
-                .await?;
+                .await
+                .with_context(|| "Certificates Hash Migrator can not create certificate")?;
```

Comment:
Can you add the certificate hash in the error context, this can be useful to identify which one is failing